nucleus/common/scattered-archive-api/src/test/java/org/glassfish/embeddable/archive/ScatteredArchiveTest.java

```diff
@@ -104,14 +104,10 @@ public void addCurrentClasspathUsingPredicate(String classpath, String excludesI
         archive.addCurrentClassPath(exclude);
 
         // THEN
-        try {
-            assertAll("classpath",
-                    () -> assertThat("Number of classpath elements " + archive.classpathElements,
-                            archive.classpathElements, hasSize(expectedElementsInClasspath))
-            );
-        } catch (AssertionError e) {
-            throw e;
-        }
+        assertAll("classpath",
+                () -> assertThat("Number of classpath elements " + archive.classpathElements,
```

Comment:
... and now we don't even need the assertAll, right? Just the assertThat is enough.

Reply:
You're right :) I mage the change quickly. I'll remove it too.

Reply:
I decided to refactor the tests and add one more assertion. In this case, assertAll makes sense because it includes 2 assertions.